lib/pa11y.js

```diff
@@ -114,42 +114,47 @@ async function runPa11yTest(url, options, state) {
 		state.autoClosePage = true;
 	}
 
-	// Intercept page requests, we need to do this in order
-	// to set the HTTP method or post data
-	await page.setRequestInterception(true);
-
-	// Intercept requests so we can set the HTTP method
-	// and post data. We only want to make changes to the
-	// first request that's handled, which is the request
-	// for the page we're testing
-	let interceptionHandled = false;
-	page.on('request', interceptedRequest => {
-		const overrides = {};
-		if (!interceptionHandled) {
-
-			// Override the request method
-			options.log.debug('Setting request method');
-			overrides.method = options.method;
-
-			// Override the request headers (and include the user-agent)
-			options.log.debug('Setting request headers');
-			overrides.headers = {
-				'user-agent': options.userAgent
-			};
-			for (const [key, value] of Object.entries(options.headers)) {
-				overrides.headers[key.toLowerCase()] = value;
+	// Until the `page.setRequestInterception` is buggy,
```

Comment:
Let's remote "Until" and add links to the specific bugs for `page.setRequestInterception` which we are working around.